targets/TARGET_Realtek/TARGET_AMEBA/TARGET_MCU_RTL8710A/device/TOOLCHAIN_ARM_STD/rtl8195a.sct

```diff
@@ -0,0 +1,71 @@
+; Realtek Semiconductor Corp.
+;
+; RTL8195A ARMCC Scatter File
+;
+; MEMORY
+; {
+;   SROM (rx)    : ORIGIN = 0x10000000, LENGTH = 0x00007000
+;   SRAM (rwx)   : ORIGIN = 0x10007000, LENGTH = 0x00070000 - 0x00007000
+;   TCM  (rwx)   : ORIGIN = 0x1FFF0000, LENGTH = 0x00010000
+;   DRAM (rwx)   : ORIGIN = 0x30000000, LENGTH = 2M
+; }
+
+LR_IRAM 0x10007000 (0x70000 - 0x7000) {
+
+  IMAGE2_TABLE 0x10007000 FIXED {
+    *rtl8195a_init*.o(.image2.ram.data*, +FIRST)
+    *rtl8195a_init*.o(.image2.validate.rodata*)
+  }
+
+  LR_IROM1 +0 FIXED {
+    *(.ARM.exidx)
+    *(.init_array)
+    *rtl8195a_crypto*.o (+RO)
+    *(i.mbedtls*)
+    *libc.a (+RO)
+    *rtx_*.o (+RO)
+    *main*.o (+RO)
+    *lib_peripheral_mbed_arm.ar (+RO)
+    *_api*.o (+RO)
+  }
+
+  RW_IRAM1 +0 UNINIT FIXED {
+    *rtl8195a_crypto*.o(+RW)
+    *libc.a (+RW)
+    *main*.o (+RW)
+    *lib_peripheral_mbed_arm.ar (+RW)
+    *_api*.o (+RW)
+    *rtl8195a_crypto*.o(+ZI)
+    *libc.a (+ZI)
+    *main*.o (+ZI)
+    *lib_peripheral_mbed_arm.ar (+ZI)
+    *_api*.o (+ZI)
+    *mbed_boot*.o (+ZI)
+  }
+
+  ARM_LIB_STACK (0x10070000 - 0x1000) EMPTY 0x1000 {
```

Comment:
```suggestion
  ARM_LIB_STACK 0x10070000 EMPTY -0x1000 {
```

Reply:
Also the stack size for Mbed 5 is 0x400. 0x1000 is needed for Mbed 2

Reply:
@deepikabhavnani Thank you for the review, I'll check your guide ASAP.